x-pack/plugins/secops/public/components/draggables/index.test.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+import { mountWithIntl, shallowWithIntl } from 'test_utils/enzyme_helpers';
+
+import { TestProviders } from '../../mock';
+
+import { DefaultDraggable, DraggableBadge } from '.';
+
+describe('draggables', () => {
+  describe('rendering', () => {
+    test('it renders the default DefaultDraggable', () => {
+      const wrapper = shallowWithIntl(
+        <DefaultDraggable
+          id="draggable-id"
+          field="some-field"
+          value="some-value"
+          queryValue="some-query-value"
+        >
+          <span>A child of this</span>
+        </DefaultDraggable>
+      );
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    test('it renders the default Badge', () => {
+      const wrapper = shallowWithIntl(
+        <DraggableBadge
+          id="draggable-id"
+          field="some-field"
+          value="some-value"
+          queryValue="some-query-value"
+          iconType="number"
+        >
+          <span>A child of this</span>
+        </DraggableBadge>
+      );
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+  });
+
+  describe('DefaultDraggable', () => {
+    test('it works with just an id, field, and value and is the default', () => {
```

Comment:
perhaps this is an outdated test title (because `value="some value"` on https://github.com/elastic/kibana/pull/33370/files#diff-c73b70d456195cc298b62951694789a2R51)?